code/.prettierrc

```diff
@@ -1,20 +1,21 @@
-{
-  "plugins": ["@trivago/prettier-plugin-sort-imports"],
-  "importOrder": [
-    "^(.*)react(.*)$",
-    "<THIRD_PARTY_MODULES>",
-    "^@/App$",
-    "^@/layouts/(?!.*\\.scss$)(.*)$",
-    "^@/components/(?!.*\\.scss$)(.*)$",
-    "^@/(?!.*\\.scss$)(.*)$",
-    "^.+\\.scss$",
-    "^[./]"
-  ],
-  "importOrderSeparation": true,
-  "importOrderSortSpecifiers": true,
-  "semi": true,
-  "trailingComma": "none",
-  "singleQuote": false,
-  "jsxSingleQuote": false,
-  "arrowParens": "always"
-}
+{
+  "plugins": ["@trivago/prettier-plugin-sort-imports"],
+  "importOrder": [
+    "^(.*)react(.*)$",
+    "<THIRD_PARTY_MODULES>",
+    "^@/App$",
+    "^@/containers/(?!.*\\.scss$)(.*)$",
+    "^@/layouts/(?!.*\\.scss$)(.*)$",
```

Comment:
Move layouts above the containers. It is more global things

Reply:
Fixed